### PR TITLE
multiply array element with a constant value

### PR DESCRIPTION
**Pull request by NidhitheCoder**

### CODE REVIEW

File: js-exercises/007_arrayAndConstant.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const readline = require("readline-sync");
+
```
Comment (kuism) [inline js-exercises/007_arrayAndConstant.js:2]: write it as a proper function

File: js-exercises/007_arrayAndConstant.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+let arr = [];
+let result = [];
+arr = readline.question("Enter array decimal elements separated by comma:");
+
+let constant = readline.question("Enter constant value :");
+const addConstant = (arr, constant) => {
+  result = arr.split(",").map(el => {
+    return constant * parseInt(el);
```
Comment (kuism) [inline js-exercises/007_arrayAndConstant.js:11]: Why you are multiplying

Comment (NidhitheCoder) [reply js-exercises/007_arrayAndConstant.js:11]: it's my mistake by giving name of PR.it is actually it is Multiply array element with a constant value.

Comment (kuism) [reply js-exercises/007_arrayAndConstant.js:11]: change the name of the function then

**Pull request merged**